src/lib/battery/battery.h

```diff
@@ -130,15 +141,21 @@ class Battery : public ModuleParams
 
 private:
 	void sumDischarged(const hrt_abstime &timestamp, float current_a);
-	void estimateStateOfCharge(const float voltage_v, const float current_a, const float throttle);
+	void estimateStateOfCharge(const float voltage_v, const float current_a);
 	uint8_t determineWarning(float state_of_charge);
 	void computeScale();
 	float computeRemainingTime(float current_a);
 
+	uORB::Subscription _actuator_controls_0_sub{ORB_ID(actuator_controls_0)};
 	uORB::PublicationMulti<battery_status_s> _battery_status_pub{ORB_ID(battery_status)};
 
+	bool _connected{false};
+	const uint8_t _source{};
```

Comment:
It's better not to initialize these here, so each constructor is forced to initialize.
```suggestion
	const uint8_t _source;
```

Reply:
Today I learned 👍 8a42aef